src/CameraOnvif.cpp

```diff
@@ -24,7 +24,10 @@ struct CameraOnvif::Private {
     _trt__GetVideoEncoderConfigurationOptionsResponse video_options;
     Private() = default;
     ~Private(){
-        // free all deserialized and managed data, we can still reuse the context and proxies after this
+        /*
```

Comment:
You are in the destructor for Private, which will have to delete all of this anyways. These comments don't make a lot of sense to me in this context.